### DISCUSSION TITLE
Typical Use case, limitations?

### DISCUSSION THREAD

**Discussion by trandana** [AndreaCatania/RuntimeSkeletalMeshGenerator#18] [General]
I'm wondering what this is typically used for?
Can you build a new skeletal mesh component based off an animated skeletal mesh at runtime? Will it take changes in vert positions into account when building?  Or only build from a skeleton in its default base pose? 

Thanks!

**Comment by Fransferdy** [comment]
I was going to procedurally generate monsters, like the spore game. Then I would use procedural animation on top of that, I have seen some people complain that animations imported with this plugin are not working in packaged game. So mostly you get the skeletal mesh and its bones.

In my case I pass it in blueprints to a PoseableMesh, and I get handles to animate each bone procedurally. 
I kind of stopped my adventure there, but the idea would be to follow some Assimp tutorials on how to apply animations to bones (in Assimp world) and apply that knowledge in my procedural animation system.

From there on you have your procedural skeletal meshes, in the amazing world of UE5, in my view that is kind of a big deal.

If you plan to use this, I recommend trying my Fork, because I fixed major bugs in there.

**Comment by acdodgen** [comment]
Does this plugin handle merging physics assets or morph targets?

**Reply by Fransferdy** [reply]
No, and it is full of bugs, don't waste your time, try this instead:

https://github.com/rdeioris/glTFRuntime?tab=readme-ov-file

**Reply by acdodgen** [reply]
This looks interesting but it is not well documented regarding the merging part, and it seems like you can't merge meshes from existing UAssets. How did you get the merging working with it?
